resources/js/hooks/use-bank-provider.ts

```
@@ -1,15 +1,49 @@
+ * - Tracks institution selection for popularity-based sorting
+ * - Non-blocking tracking that won't disrupt user experience
+ * - Extends provider-specific hooks with tracking capabilities
+ *
+ * @param props - Configuration including institution id, name, and provider type
+ * @returns Extended provider hook with tracking functionality
+ */
+export function useBankProvider({ provider, id }: ConnectBankProviderProps) {
     switch (provider) {
         case "teller":
-            return useTellerConnect();
+            const tellerConnect = useTellerConnect();
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)